.travis.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     # GitHub Pages Deployment: https://docs.travis-ci.com/user/deployment/pages/
     - stage: Docs
       python: "3.7"
-      if: type != cron
+      if: type != cron AND branch = master
```

Comment:
I'm not sure if this worked. See current travis CI report where we still build the docs.